### PR TITLE
Fixed issue: Value was not being set when selecting with a mouse

### PR DESCRIPTION
**Pull request by luaybs**
This code was removed when it was forked, I noticed that it was bugging out for the single value select tags. I re-added it and it worked.

### CODE REVIEW

**Comment by rdawson-hmrc** [comment]
Please merge this, I have seen the same issue when selecting values with the mouse and this fixes it. Thanks @luaybs

**Comment by luaybs** [comment]
@rdawson-hmrc Do you have any idea who I should add as a reviewer from the maintainers to check this out?

**Comment by rdawson-hmrc** [comment]
@luaybs I don't know, @ryanb-gds seems to be the last person to make changes and @tvararu is top of the committers list so tagging them here, maybe one will respond

**Review by tvararu — CHANGES_REQUESTED** [review]
Hi! I indeed built the original `accessible-autocomplete`. I no longer work on it. I also wasn't aware of this sibling project. But I can offer my own 2 cents. 👍 

The commit that removed this event handler is this: https://github.com/alphagov/accessible-autocomplete-multiselect/commit/6aef4e705703f488ba7dc67e36f5c61866227b40#diff-012dea2ebaecdc6abc60666b102a2c9ea108885579906e97b90bc08d0a29d210L294

That same commit also includes git `<<<<<<< HEAD` merge artefacts which I thought was odd: https://github.com/alphagov/accessible-autocomplete-multiselect/commit/6aef4e705703f488ba7dc67e36f5c61866227b40#diff-012dea2ebaecdc6abc60666b102a2c9ea108885579906e97b90bc08d0a29d210R612

It was introduced via this PR which explains that a big merge had to be done on this project: https://github.com/alphagov/accessible-autocomplete-multiselect/pull/45

The likely explanation seems to be that this code was lost in the cracks during the merge.

So this code looks good to me, since it's code that already exists in the main `accessible-autocomplete` https://github.com/alphagov/accessible-autocomplete/blob/79370c8c66f7496fb503bb1840cd590157f97232/src/autocomplete.js#L298 (and I remember the context for this bug, it breaks mobile Safari)

I did try running this branch locally but failed to install Node `14.18.2`. Looks like the main autocomplete is using `lts/iron` now.

This PR is incomplete, because it doesn't update the `dist`, the actual artefacts that get pushed to `npm` and the other package managers.

In a nutshell you need to `npm i && npm run build` and commit that to this PR for it to be complete. More details in `CONTRIBUTING`: https://github.com/alphagov/accessible-autocomplete-multiselect/blob/main/CONTRIBUTING.md#prepush-checks

If you do manage to get that sorted, I think this would be sensible to merge. Not sure who is around to assist, but @36degrees might be able to point to someone...

**Comment by luaybs** [comment]
Sure thing, will run the build tomorrow and update the PR! Thanks for your input, appreciate it

**Comment by luaybs** [comment]
@tvararu done!

**Review by tvararu — APPROVED** [review]
Ran this locally on mobile Safari. I can confirm that tapping options selects them successfully.

**Review by ryanb-gds — APPROVED** [review]
Thank you for fixing this
